Resources/About Contents.html

```diff
@@ -1,17 +1,17 @@
 <html>
 <head>
 </head>
-<body style="background-color: #1f1f1f">
+<body style="background-color: BACKGROUNDCOLOR">
 <style type="text/css">
-    h3 {color: #aaaaaa}
-    h5 {text-decoration: plain; color: #aaaaaa}
-    A:link {text-decoration: underline; color: #aaaaaa}
-    A:visited {text-decoration: none; color: #aaaaaa}
-    A:active {text-decoration: none; color: #aaaaaa}
+    h3 {color: TEXTCOLOR}
+    h5 {text-decoration: plain; color: TEXTCOLOR}
+    A:link {text-decoration: underline; color: TEXTCOLOR}
+    A:visited {text-decoration: none; color: TEXTCOLOR}
+    A:active {text-decoration: none; color: TEXTCOLOR}
     body { font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
            font-weight: 300; }
 </style>
-<div style="text-align:left; border:solid transparent 1px; padding:0.5em 1em 0.5em 1em; overflow:auto; font-size:10pt; font-family:Helvetica; color:#aaaaaa"; A:link {text-decoration: underline; color:#aaaaaa}>
+<div style="text-align:left; border:solid transparent 1px; padding:0.5em 1em 0.5em 1em; overflow:auto; font-size:10pt; font-family:Helvetica; color:TEXTCOLOR"; A:link {text-decoration: underline; color:TEXTCOLOR}>
```

Comment:
I have no idea how the 

```
A:link {text-decoration: underline; color:TEXTCOLOR}
```

part can work here, this looks very invalid.

Reply:
it actually get's replaced: TEXTCOLOR by [htmlContent replaceOccurrencesOfString:@"TEXTCOLOR" withString:textColor options:NSLiteralSearch range:NSMakeRange(0, [htmlContent length])];

Reply:
Yes I know, it is still invalid, as it is not inside the style and you can't do stuff like `a:link { … }` even in a style attribute.

Reply:
okay I don't know enough about html here. Let's address that in a follow up! :)